client/src/views/devices/ExportRecoveryDevicePage.vue

```diff
@@ -224,15 +226,14 @@ async function downloadRecoveryKey(): Promise<void> {
 
 async function downloadRecoveryFile(): Promise<void> {
   fileDownload(file, t('ExportRecoveryDevicePage.filenames.recoveryFile', { org: orgId.value }));
-  setTimeout(() => {
-    recoveryFileDownloaded.value = true;
-    notificationManager.showToast(
-      new Notification({
-        message: t('ExportRecoveryDevicePage.toasts.fileDownloadOk'),
-        level: NotificationLevel.Success,
-      }),
-    );
-  }, 500);
+  recoveryFileDownloaded.value = true;
```

Comment:
Why removing the `setTimeout()`?

Reply:
It was set for waiting the file to be downloaded?

Reply:
Yep, it's here to simulate the download time. Since we don't really have a way to tell whether the file was really downloaded or not, we added this little thing so that the appearance does not change immediately.